vector/src/test/java/im/vector/app/test/fakes/FakeSharedSecretStorageService.kt

```diff
@@ -75,10 +75,15 @@ class FakeSharedSecretStorageService : SharedSecretStorageService by mockk() {
 
     override fun checkShouldBeAbleToAccessSecrets(secretNames: List<String>, keyId: String?) = integrityResult
 
+    @Deprecated("Requesting custom secrets not yet support by rust stack, prefer requestMissingSecrets")
```

Comment:
I do not think it is useful to have the `@Deprecated` here. It's in the interface, and for Fake we do not really care.

Reply:
see above